docs/reference/items.md

```diff
@@ -908,6 +908,112 @@ const result = await client.request(
 </template>
 </SnippetToggler>
 
+## Update Items by Query
+
+Update multiple items by selecting what items to update using a Directus query object.
```

Comment:
This section should be updated to be `update_<collection>_batch` operation as `update_<collection>_by_query` is not implemented. The `update_<collection>_batch` GQL operation utilises `ItemsService.updateBatch` under the hood.

Reply:
That's the other way around yeah? It shouldn't use batch but should use by_query instead as per #24101

Reply:
Yup, it'll be the other way around. 👍